### PR TITLE
implement CategoryItem component

### PR DESCRIPTION
**Pull request by tranchinh123**

### CODE REVIEW

File: bookshelter-app/src/components/CategoryItem/index.tsx [hunk 1]
```diff
@@ -1,5 +1,18 @@
-const CateGoryItem = () => {
-  return <></>;
+import styles from './index.module.css';
+interface CateGoryItemProps {
+  category: string;
+  quantity: number;
+}
+const CateGoryItem = ({ category, quantity }: CateGoryItemProps) => {
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/CategoryItem/index.tsx]: should be BookCategory

File: bookshelter-app/src/components/CategoryItem/index.tsx [hunk 2]
```diff
@@ -1,5 +1,18 @@
-const CateGoryItem = () => {
-  return <></>;
+import styles from './index.module.css';
+interface CateGoryItemProps {
+  category: string;
+  quantity: number;
+}
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/CategoryItem/index.tsx]: add an empty line, pls similar check for all

File: bookshelter-app/src/components/CategoryItem/index.tsx [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+const CateGoryItem = ({ category, quantity }: CateGoryItemProps) => {
+  return (
+    <>
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/CategoryItem/index.tsx]: why need <></>

Comment (tranchinh123) [reply bookshelter-app/src/components/CategoryItem/index.tsx]: Fixed

File: bookshelter-app/src/components/CategoryItem/index.tsx (hunk 2 above)
Comment (tranchinh123) [reply bookshelter-app/src/components/CategoryItem/index.tsx]: fixed

File: bookshelter-app/src/components/CategoryItem/index.tsx (hunk 1 above)
Comment (tranchinh123) [reply bookshelter-app/src/components/CategoryItem/index.tsx]: Fixed

**Pull request merged**